src/client-agent/config.c

```diff
@@ -128,14 +128,26 @@ cJSON *getLabelsConfig(void) {
 
     unsigned int i;
     cJSON *root = cJSON_CreateObject();
-    cJSON *labels = cJSON_CreateObject();
+    cJSON *label_s = cJSON_CreateArray();
+    cJSON *label = cJSON_CreateObject();
+    cJSON *labels = cJSON_CreateArray();
 
     if (agt->labels) {
         for (i=0;agt->labels[i].key;i++) {
-            cJSON_AddStringToObject(labels,agt->labels[i].key,agt->labels[i].value);
+            cJSON *label_k = cJSON_CreateObject();
+            cJSON_AddStringToObject(label_k,"value", agt->labels[i].value);
```

Comment:
Add spaces between parameters as in `param1, param2, ..., paramN`. Applies to the whole PR.